src/v/debug_bundle/tests/debug_bundle_service_test.cc

```diff
@@ -38,11 +40,27 @@ struct debug_bundle_service_fixture : public seastar_test {
         co_await _service.start(_data_dir);
     }
 
-    ss::future<> TearDownAsync() override { co_await _service.stop(); }
+    ss::future<> TearDownAsync() override {
+        if (stop_service) {
+            co_await _service.stop();
+        }
+    }
 
     std::filesystem::path _rpk_shim_path;
     std::filesystem::path _data_dir;
     ss::sharded<debug_bundle::service> _service;
+    bool stop_service = true;
+};
+
+struct debug_bundle_service_started_fixture
+  : public debug_bundle_service_fixture {
+    ss::future<> SetUpAsync() override {
+        co_await debug_bundle_service_fixture::SetUpAsync();
+        ASSERT_NO_THROW_CORO(co_await _service.invoke_on_all(
+          [](debug_bundle::service& s) -> ss::future<> {
+              co_await s.start();
+          }));
```

Comment:
q(sanity check): asserting that you can double-call `start` w/o issue?

Reply:
I don't think I'm calling start twice.  There's two fixtures, one that doesn't automatically call start (which is used by a couple of tests) and this fixture that does call start as part of set up.